src/main/java/com/points/calculate/NumberNode.java

```diff
@@ -0,0 +1,11 @@
+package com.points.calculate;
+
+import com.points.node.twochild.Node;
+
+public class NumberNode extends Node {
+	
+	public NumberNode(int num) {
```

Comment:
This is probably a NumberNode<Integer> implementation where you could generalize it a bit more.